### ISSUE TITLE
Should we adopt a style guide?

### ISSUE THREAD

**Issue by scotttrinh** [RemoteTogether/fido#1]
My vote is yes, [the Airbnb one](https://github.com/airbnb/javascript). Should help with having so many collaborators.

**Comment by scotttrinh** [comment]
[Here](https://github.com/airbnb/javascript/tree/master/es5) is the link to the Airbnb ES5 Style Guide, the other link I posted was to the ES2015 style guide. I suppose it's a separate question as to what version of Node we want to target?

**Comment by josefdlange** [comment]
I think _any_ style guide is a good one, so long as we keep to it. Are there available style linters for the aforementioned styles?

**Comment by scotttrinh** [comment]
Yeah, Airbnb has [eslint](https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb) and [jshint](https://github.com/airbnb/javascript/blob/master/linters/.jshintrc) linters available.
